packages/agent/src/agent/http/index.ts

```diff
@@ -392,6 +420,14 @@ export class HttpAgent implements Agent {
     return this.rootKey;
   }
 
+  public invalidateIdentity(): void {
+    this._identity = null;
+  }
+
+  public replaceIdentity(identity: Identity): void {
```

Comment:
Is this used only for testing purposes? or is this part of the API to be used somewhere else?

Reply:
I'm also not sure about this: what would `replaceIdentity` be used for?

Reply:
If someone wants to continue using an agent hat has been invalidated, this will allow them to update the same instance with a fresh identity without refreshing the page or re-instantiating a new HttpAgent.

Both of those options are still viable, but this will provide additional flexibility